### PR TITLE
Make sure to refresh page after restart and update

### LINKED ISSUE #1142: Reload ASF-ui page after ASF update
## Description
After successful ASF update ASF-ui page in browser does not auto reload. Probably hard reload is needed (CTRL+F5).

## Steps to reproduce
1. Open ASF-ui page in browser,
2. Make sure there is ASF update available,
3. Update ASF either way (interactive console, chat, ASF-ui),
4. ASF updated and restarted itself as well as started new IPC server (see on attached screenshot local version is 4.3.1.1), but ASF-ui page was not reloaded (see bottom right corner on attached screenshot).

## Purpose

Reload ASF-ui page in browser to update web interface after successful ASF update and subsequent ASF restart, which implies that IPC server was also restarted.

## Screenshots
chrome_9hwWoPaNb8

### LINKED ISSUE #1211: ASF-ui does not reload page after restart
## Description
After [#2057](https://github.com/JustArchiNET/ArchiSteamFarm/issues/2057) is closed, ASF-ui doesn't reload browser page anymore after ASF restart.

## Steps to reproduce
1. Install ASF [this version](https://github.com/JustArchiNET/ArchiSteamFarm/actions/runs/425550701) or above,
2. Open ASF-ui,
3.  Restart ASF via restart button in ASF-ui.

## Screenshots
image

### PR DESCRIPTION
**Pull request by MrBurrBurr**
Fixes #1211 
Fixes #1142 

@Botan626 Feel free to test.

### CODE REVIEW

**Comment by JustArchi** [comment]
@MrBurrBurr I've fixed CI for this, it looks like GitHub had some hiccup.

File: src/components/AsfReleases.vue [hunk 1]
```diff
@@ -85,17 +87,31 @@
...
-          this.$info(this.$t('restart-initiated'));
+        try {
+          await this.$http.post('asf/update');
           await waitForRestart();
-          this.$success(this.$t('restart-complete'));
+          this.$success(this.$t('update-complete'));
+          await delay(3000);
           window.location.reload();
+        } catch (err) {
+          if (err.message === 'HTTP Error 504' || err.message === 'Network Error') {
```
Comment (JustArchi) [inline src/components/AsfReleases.vue:103]: There isn't a better way to do this?

**Comment by Botan626** [comment]
ASF-ui didn't refresh its page after executing 'restart' command in both interactive console and 

<details>
  <summary>Commands page in AF-ui.</summary>

image

</details>

**Comment by Botan626** [comment]
Also ASF-ui page wasn't reloaded after 'restart' command was sent via chat.

PR works if restart was performed via restart button in ASF-ui.

**Comment by Botan626** [comment]
Updating via install badge and update button is ok, ASF-ui page reloads itself.

If updating by executing 'update' command in Commands page in ASF-ui, the page won't reload, there is no even notification 'New version is available! Trying to update...'.

**Comment by Botan626** [comment]
ASF-ui page wasn't reloaded after updating via interactive console, I guess the same result will be after updating via Steam chat, but didn't try it.

<details>
<summary>Screenshot</summary>

image

</details>

File: src/components/AsfReleases.vue (hunk 1 above)
Comment (MrBurrBurr) [reply src/components/AsfReleases.vue:103]: I am sure there is a better way, but I can't find one at the moment. Maybe @Aareksio can help out?

**Comment by MrBurrBurr** [comment]
Thank you for testing @Botan626.

I will wait for @Aareksio and see if I can catch his response before merging.

File: src/components/AsfReleases.vue (hunk 1 above)
Comment (MrBurrBurr) [reply src/components/AsfReleases.vue:103]: I may found a fix for this mess. It will be implemented with #1180

**Pull request merged**